src/sonic-config-engine/minigraph.py

```diff
@@ -1450,7 +1450,8 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         'hostname': hostname,
         'hwsku': hwsku,
         'type': device_type,
-        'synchronous_mode': 'enable'
+        'synchronous_mode': 'enable',
+        'yang_config_validation': 'disable'
```

Comment:
Please create another PR for sonic-config-engine, and update unit test for sonic-config-engine.

Reply:
Created separate PR